framework/components/ecl-footers/_footers.scss

```diff
@@ -1,142 +1,122 @@
-/*
-Footer
-
-Site-wide footer component
-
-Markup: html/footer.html
-
-Style guide footer
-*/
-
-.footer {
-  background-color: $blue-medium;
+/**
+ * Footers component
+ * @define footers ; weak
+ */
+
+.ecl-footers__site-identity,
+.ecl-footers__site-corporate,
+.ecl-footers__ec {
+  @extend %hidden-print;
 }
 
-.footer-bottom {
-  background-color: $brand-primary;
-  padding: 0.9em 0;
+.ecl-footers__social-icon {
+  &::before {
+    margin-right: 8px !important;
+  }
+
+  &::after {
+    content: none !important;
```

Comment:
I decided to keep this, as it's in the current codebase of europa-theme and I can't also understand it to know whether it's save to remove.